### PR TITLE
gui: Update bitcoin_sv.ts

### PR DESCRIPTION
**Pull request by ghost**
Added and confirmed 50+ translations

### CODE REVIEW

File: src/qt/locale/bitcoin_sv.ts [hunk 1]
```diff
@@ -422,96 +422,92 @@ This product includes software developed by the OpenSSL Project for use in the O
...
-?} {2
-?} {3
-?} {4?}</translation>
+Adress: %4</translation>
     </message>
     <message>
         <location line="+446"/>
         <source>Staking.&lt;br&gt;Your weight is %1&lt;br&gt;Network weight is %2&lt;br&gt;&lt;b&gt;Estimated&lt;/b&gt; time to earn reward is %3.</source>
-        <translation type="unfinished"></translation>
+        <translation>Staking.&lt;br&gt;Din vikt är %1&lt;br&gt;Nätverkets vikt är %2&lt;br&gt;&lt;b&gt;Beräknad&lt;/b&gt; tid till premie är %3.</translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: Alternatively: "Beräknad tid till utdelning är"

File: src/qt/locale/bitcoin_sv.ts [hunk 2]
```diff
@@ -679,7 +675,7 @@ Scraper(s) not publishing: %5.</source>
     <message>
         <location line="+83"/>
         <source>%1 active connection(s) to Gridcoin network</source>
-        <translation type="unfinished">%1 aktiv anslutning till Gridcoin nätverket</translation>
+        <translation>%1 aktiva anslutningar till Gridcoin nätverket</translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: Gridcoin-nätverket

File: src/qt/locale/bitcoin_sv.ts [hunk 3]
```diff
@@ -738,17 +734,17 @@ Scraper(s) not publishing: %5.</source>
     <message>
         <location line="+18"/>
         <source>Wallet is &lt;b&gt;encrypted&lt;/b&gt; and currently %1 </source>
-        <translation type="unfinished"></translation>
+        <translation>Plånboken är &lt;b&gt;krypterad&lt;/b&gt; och förtillfället %1 </translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: förtillfället -> för tillfället

File: src/qt/locale/bitcoin_sv.ts [hunk 4]
```diff
@@ -1156,72 +1152,72 @@ Detta betyder att en avgift på minst %2 krävs.</translation>
...
         <location line="-157"/>
         <source>Verify CPID is in Neural Network</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifiera att CPID finns i nätverket</translation>
     </message>
     <message>
         <location line="+51"/>
         <source>Verify BOINC path</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifiera BOINC sökväg</translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: BOINC-sökväg. I think I missed a couple of these higher up :)

File: src/qt/locale/bitcoin_sv.ts [hunk 5]
```diff
@@ -1156,72 +1152,72 @@ Detta betyder att en avgift på minst %2 krävs.</translation>
...
         <location line="+51"/>
         <source>Find PrimaryCPID (Windows Only)</source>
-        <translation type="unfinished"></translation>
+        <translation>Hitta primär CPID (Endast Windows)</translation>
     </message>
     <message>
         <location line="+41"/>
         <source>Verify clock</source>
-        <translation type="unfinished"></translation>
+        <translation>Verifiera tid</translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: I think we want "klocka" instead of "tid" here.

File: src/qt/locale/bitcoin_sv.ts [hunk 6]
```diff
@@ -2673,7 +2669,7 @@ Detta betyder att en avgift på minst %2 krävs.</translation>
     <message>
         <location line="+66"/>
         <source>Send Custom Message to a Gridcoin Recipient</source>
-        <translation type="unfinished"></translation>
+        <translation>Skicka ett meddelande till en Gridcoin mottagare</translation>
```
Comment (denravonska) [inline src/qt/locale/bitcoin_sv.ts]: Gridcoin-mottagare :)

**Review by denravonska — APPROVED** [review]
Approved with minor suggestions. Refreshing to see a Swedish translation which isn't riddled with särskrivningar.

File: src/qt/locale/bitcoin_sv.ts (hunk 3 above)
Comment (ghost) [reply src/qt/locale/bitcoin_sv.ts]: My bad!

File: src/qt/locale/bitcoin_sv.ts (hunk 4 above)
Comment (ghost) [reply src/qt/locale/bitcoin_sv.ts]: Well there are more phrases to translate. Will do some more in a month or two. Can go back and search for the ones not corrected.

File: src/qt/locale/bitcoin_sv.ts (hunk 6 above)
Comment (ghost) [reply src/qt/locale/bitcoin_sv.ts]: Apperently you find some särskrivningar.

**Comment by jamescowens** [comment]
@sweede-se Can you make the appropriate changes and/or indicate to close-out the conversations above and then put up the commit and I will merge. Thanks

**Comment by ghost** [comment]
@jamescowens I edited according to @denravonska s suggestions. Also double checked my own work and all previous translations. 

Do I need to do something else? I'm a total Github newbie.

**Comment by jamescowens** [comment]
Nope. Good!

**Pull request merged**